### ISSUE TITLE
Chart & kendo-chart-category-axis doesn't format the date in the correct language

### ISSUE THREAD

**Issue by EricSch** [telerik/kendo-angular#569]
When I'm using the grid category with type "date", the shown formatted date doesn't respect the language of the Intl Service. It shows always the English format (5/11 instead of 11.5).

```
<kendo-chart-category-axis>
      <kendo-chart-category-axis-item [categories]="categories" baseUnit="fit" maxDateGroups="30" type="date">
       </kendo-chart-category-axis-item>
</kendo-chart-category-axis>
```

Thanks

**Comment by danielkaradachki** [comment]
Indeed, the default date axis formats currently do not depend on the locale. We will fix this.

Not sure if I understand the issue with the Grid. Please clarify.

**Comment by EricSch** [comment]
Sorry, was my mistake with 'Grid' in the title, I corrected it.

**Comment by tsvetomir** [comment]
The bug should be fixed in v1.1.1 of the kendo-angular-charts package
